practices/practice-one/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,73 @@
+import { Link } from "react-router-dom";
+import cinndy from "assets/images/Cinndy.jpg";
+import cector from "assets/icons/Vector.svg";
+import Heading from "components/Heading/Heading";
+import Button from "components/Button/Button";
+import Paragraph from "components/Paragraph/Paragraph";
+import Input from "components/Input/Input";
+
+import "./header.css";
+
+function Header() {
```

Comment:
dont put it in component, move to constants instead

Reply:
yes sir